.github/actions/env/action.yml

```diff
@@ -24,22 +24,23 @@ runs:
         auto-update-conda: true
         python-version: ${{ inputs.python-version }}
 
-    - if: ${{ contains(fromJson('["lts", "nightly"]'), inputs.pytorch-version ) }}
-      name: Install dependencies
+    - name: Install curl via conda
       shell: bash -l {0}
-      run: |
-        conda install curl -c conda-forge
-        conda install pytorch cpuonly -c pytorch-${{ inputs.pytorch-version }}
-        pip install .[dev,x]
+      run: conda install curl -c conda-forge
 
+    - if: ${{ contains(fromJson('["lts", "nightly"]'), inputs.pytorch-version ) }}
+      name: Install pytorch
+      shell: bash -l {0}
+      run: conda install pytorch cpuonly -c pytorch-${{ inputs.pytorch-version }}
 
     - if: ${{ contains(fromJson('["lts", "nightly"]'), inputs.pytorch-version ) == false}}
-      name: Install dependencies
+      name: Install pytorch
       shell: bash -l {0}
-      run: |
-        conda install curl -c conda-forge
-        conda install pytorch=${{ inputs.pytorch-version }} cpuonly -c pytorch
-        pip install .[dev,x]
+      run: conda install pytorch=${{ inputs.pytorch-version }} cpuonly -c pytorch
+
+    - name: Install Kornia dev
+      shell: bash -l {0}
+      run: pip install .[dev,x]
```

Comment:
@johnnv1 seems that editable mode was the issue with accelerate @Mueller

Reply:
Weird, my first commit had failed with the same error (and it was not in editable mode)
- https://github.com/kornia/kornia/actions/runs/3576469567/jobs/6014319026#step:3:389
- https://github.com/kornia/kornia/actions/runs/3576469567/jobs/6014319026#step:4:9618

Reply:
This keeps happening sometimes in main

Today at the scheduled tests on all macOS with torch 1.12.1:
- fail: https://github.com/kornia/kornia/actions/runs/3945580309/jobs/6752600289#step:4:9870
- but have been installed: https://github.com/kornia/kornia/actions/runs/3945580309/jobs/6752600289#step:3:788